server.js

```diff
@@ -21,6 +22,7 @@ client.on('error', errorHandler);
 
 let lat = 0;
 let lng = 0;
```

Comment:
Is this DRY code? Could you define your variables when you are first using them?